assets/src/edit-story/app/prepublish/constants.js

```diff
@@ -14,8 +14,311 @@
+      ),
+    },
+    MISSING_VIDEO_ALT_TEXT: {
+      MAIN_TEXT: __('Video is missing assistive text', 'web-stories'),
+      HELPER_TEXT: __(
+        'Add meaningful “alt” text to images to optimize accessibility and indexability.',
+        'web-stories'
+      ),
+    },
+  },
+  DISTRIBUTION: {
+    MISSING_DESCRIPTION: {
+      MAIN_TEXT: __('Missing story description', 'web-stories'),
+      HELPER_TEXT: __('Add a description for your story.', 'web-stories'),
+    },
+  },
+  GENERAL_GUIDELINES: {
+    STORY_TOO_SHORT: {
+      MAIN_TEXT: __('Story too short', 'web-stories'),
+      HELPER_TEXT: sprintf(
+        /* translators: 1: minimum number of pages. 2: maximum number of pages. */
+        __('Use between %1$d and %2$d pages in your story.', 'web-stories'),
+        MIN_STORY_PAGES,
+        MAX_STORY_PAGES
+      ),
+    },
+    STORY_TOO_LONG: {
+      MAIN_TEXT: __('Story too long', 'web-stories'),
+      HELPER_TEXT: sprintf(
+        /* translators: 1: minimum number of pages. 2: maximum number of pages. */
+        __('Use between %1$d and %2$d pages in your story.', 'web-stories'),
+        MIN_STORY_PAGES,
+        MAX_STORY_PAGES
+      ),
+    },
+    STORY_TITLE_TOO_LONG: {
+      MAIN_TEXT: __('Story title too long', 'web-stories'),
+      HELPER_TEXT: sprintf(
+        /* translators: 1: minimum number of story characters */
+        __('Story title should have %1$d characters or fewer.', 'web-stories'),
+        MAX_STORY_CHARACTERS
+      ),
+    },
+  },
+  TEXT: {
+    TOO_MUCH_PAGE_TEXT: {
+      MAIN_TEXT: __('Too much text on page', 'web-stories'),
+      // eslint-disable eslint/no-useless-escape
+      HELPER_TEXT: sprintf(
+        /* translators: 1: maximum number of story characters. 2: maximum percentage of characters, depending on number of pages. */
+        __(
+          'Keep text to max %1$d characters per page. Consider using a page attachment, breaking up the text into multiple screens, or keeping the total number of pages with a lot of text to less than %2$d%% percent of the pages in the story.',
+          'web-stories'
+        ),
+        MAX_STORY_CHARACTERS,
+        MAX_CHARACTER_PERCENTAGE
+      ),
+    },
+    TOO_LITTLE_TEXT: {
+      MAIN_TEXT: __('Too little text', 'web-stories'),
+      HELPER_TEXT: sprintf(
+        /* translators: 1: minimum number of story characters */
+        __(
+          'The entire story should have a minimum of %1$d characters of text in total.',
+          'web-stories'
+        ),
+        MIN_STORY_CHARACTERS
+      ),
+    },
+  },
+  MEDIA: {
+    VIDEO_IMAGE_TOO_SMALL_ON_PAGE: {
+      MAIN_TEXT: __('Video or image too small on the page', 'web-stories'),
+      HELPER_TEXT: __(
+        // todo add [more info](https://amp.dev/documentation/guides-and-tutorials/start/create_successful_stories/#visual-treat)
+        'Use full screen videos and images where possible to keep to an immersive feeling.',
+        'web-stories'
+      ),
```

Comment:
Let's say this text contains a link in the future, what would using `TranslateWithMarkup` look like in the future?